ros_gz_sim/ros_gz_sim/actions/gz_spawn_model.py

```diff
@@ -72,7 +72,7 @@ def __init__(
         self.__file = file
         self.__xml_string = xml_string
         self.__topic = topic
-        self.__name = name
+        self.__entity_name = entity_name
```

Comment:
Nit: Could you update the documentation as well?

Reply:
done